templates/web/peterborough/waste/confirmation.html

```diff
@@ -21,20 +21,38 @@ <h1 class="govuk-panel__title">
       <p>
         [% IF report.get_extra_metadata('grouped_ids') %]
           Your reference numbers are: <strong>[% report.id %], [% report.get_extra_metadata('grouped_ids').join(', ') %]</strong>.
-          If you need to contact us about this request please quote these references.
         [% ELSE %]
           Your reference number is <strong>[% report.id %]</strong>.
-          If you need to contact us about this request please quote this reference.
         [% END %]
       </p>
-      [% IF first_page == 'request' %]
-        <p>
-          Containers arrive typically within two weeks, but this may vary due to demand.
-        </p>
-      [% END %]
-      [% IF report.user.email %]
-        <p>A copy has been sent to your email address, [% report.user.email %].</p>
-      [% END %]
+    </div>
+</div>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+[% IF report.get_extra_metadata('grouped_ids') %]
+    <p>If you need to contact us about this request please quote these references.
+[% ELSE %]
+    <p>If you need to contact us about this request please quote this reference.
+[% END %]
+
+[% IF report.user.email %]
+    <p>A copy has been sent to your email address, [% report.user.email %].</p>
+[% END %]
+
+[% IF first_page == 'request' %]
+    <p>Containers arrive typically within two weeks, but this may vary due to demand.</p>
+[% END %]
+
+[% IF report.category == 'Wheels' OR report.category == 'Lid' %]
+    <p>Please leave your bin accessible on your property for us to repair.
```

Comment:
Does this (and L35/L37)  `<p>` need closing, or is HTML smart enough for that to be implied? I forget.

Reply:
Yeah, end p can be omitted if followed by another block level element or end of parent (unless the parent is an `a`!).